client/dist/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Google Shopping</title>
+</head>
+<body>
+<div id="app"></div>
+<div id="reviews"></div>
+<div id="seller"></div>
+<div id="images"></div>
```

Comment:
Be aware that this will probably have to change. Our team will need to come to an agreement on what our id's will be.